app/src/main/res/values/strings.xml

```diff
@@ -17,12 +17,12 @@
   ~ along with this program.  If not, see <https://www.gnu.org/licenses/>.
   --><resources>
     <string name="app_name">Code Scanner</string>
-    <string name="simple_description">An app to scan and create codes</string>
-    <string name="source_code">Open Source</string>
```

Comment:
The `simple_description` and `open_source_view_or_contribute` strings seem ok, but for `source_code`, I think "Open Source" or "Source Code" would be more appropriate for the header of such a button.
![image](https://user-images.githubusercontent.com/58442255/162122134-c1e560b4-6b98-4f86-820f-1736190cdb5f.png)

Reply:
Both "Open Source" and "Source Code" seem fine to me, but given that the strings have already been translated, I think that it can be left as it is.

Thanks